src/main/java/frc/robot/util/TalonFXAdapter.java

```diff
@@ -19,20 +19,23 @@
 public final class TalonFXAdapter implements MotorController {
   private final TalonFX talonFX;
   private final double metersPerRotation;
+  private final MotorOutputConfigs motorOutputConfigs;
 
   /**
    * Constructs a TalonFXAdapter.
    *
-   * <p>This private constructor is intended for use only by delegating public constructors or
-   * factory methods. It assumes the {@link TalonFX} object is already configured or will be
-   * configured appropriately by the public constructors or factory methods.
+   * <p>This constructor assumes the {@link TalonFX} object is already configured or will be
+   * configured to match the provided motor output configuration by the caller.
    *
    * @param talonFX The TalonFX object to adapt.
+   * @param motorOutputConfigs The motor output configuration.
    * @param metersPerRotation The distance in meters the attached mechanism moves per rotation of
```

Comment:
This should really be `distancePerRotation`. Since the implementation simply multiplies the number of rotations of the motor output shaft by this constant, the units are irrelevant here.